src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -24,6 +26,13 @@ public Movie get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        Session session = HibernateUtil.getSessionFactory().openSession();
+        try {
+            return session.createQuery("From Movie", Movie.class).getResultList();
```

Comment:
The `getAll` method should delegate to the `movieDao.getAll()` method instead of directly using the Hibernate session. This ensures consistency and encapsulation of data access logic within the DAO layer.